src/source/Rtp/RtpPacket.c

```diff
@@ -99,7 +117,48 @@ STATUS createRtpPacketFromBytes(PBYTE rawPacket, UINT32 packetLength, PRtpPacket
         pRtpPacket = NULL;
     }
 
-    if (pRtpPacket != NULL) {
+    if (ppRtpPacket != NULL) {
+        *ppRtpPacket = pRtpPacket;
+    }
+    LEAVES();
+    return retStatus;
+}
+
+STATUS constructRetransmitRtpPacketFromBytes(PBYTE rawPacket, UINT32 packetLength, UINT16 sequenceNum, UINT8 payloadType, UINT32 ssrc, PRtpPacket* ppRtpPacket)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PRtpPacket pRtpPacket = (PRtpPacket) MEMALLOC(SIZEOF(RtpPacket));
+    PBYTE pPayload = NULL;
```

Comment:
Very old compilers will complain about this - move this definition above any function calls